entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [[ ! "${SUPERUSER_PASSWORD}" == "" ]]; then
+    /opt/mumble/murmur.x86 -fg -ini ${CONFIG_PATH} -supw ${SUPERUSER_PASSWORD}
```

Comment:
I don't think this command needs the `-fg` option since it's not actually running mumble.

Reply:
true